README.md

```diff
@@ -23,11 +28,27 @@ npm install simple-line-icons --save
 
 ```
 
-you can also clone or download this repository as zip.
+Or, you can also clone or download this repository as zip.
 
-Preview & Docs
-===
-[https://thesabbir.github.io/simple-line-icons] (https://thesabbir.github.io/simple-line-icons)
+
+Customizing LESS/SASS variables
+====
+
+###LESS:
+
+```less
+@simple-line-font-path        : "/path/to/font/files";
+@simple-line-font-family      : "desired-name-font-family";
+@simple-line-icon-prefix      : prefix-;
+```
+
+###SASS:
+
+```sass
+$simple-line-font-path        : "/path/to/font/files";
+$simple-line-font-family      : "desired-name-font-family";
+$simple-line-icon-prefix      : prefix-;
```

Comment:
Forgot quotes on sass section. For less, its ok.

On Thu, Aug 27, 2015 at 2:48 AM, Michiel De Mey <notifications@github.com>
wrote:

> In README.md
> <https://github.com/thesabbir/simple-line-icons/pull/23#discussion_r38032933>
> :
>
> > +====
> > +
> > +###LESS:
> > +
> > +```less
> > +@simple-line-font-path        : "/path/to/font/files";
> > +@simple-line-font-family      : "desired-name-font-family";
> > +@simple-line-icon-prefix      : prefix-;
> > +```
> > +
> > +###SASS:
> > +
> > +```sass
> > +$simple-line-font-path        : "/path/to/font/files";
> > +$simple-line-font-family      : "desired-name-font-family";
> > +$simple-line-icon-prefix      : prefix-;
>
> Forgot the quotes here as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thesabbir/simple-line-icons/pull/23/files#r38032933>.
>